times_reader/transforms.py

```diff
@@ -2198,6 +2198,7 @@ def dump_tables(tables: List, filename: str) -> List:
     return tables
 
 
+# TODO: A function like this is also inluded in main.py. Should we delete one?
 def expand_rows_parallel(
     tables: List[datatypes.EmbeddedXlTable],
 ) -> List[datatypes.EmbeddedXlTable]:
```

Comment:
@siddharth-krishna @samwebster should we delete the function from here or from main?

Reply:
I don't know why there's two of these. My guess is it was copy-pasted and not moved by accident. I'd say remove this one from main.